packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -325,6 +325,9 @@ export class SingleAddressWallet implements ObservableWallet {
     this.#resubmitSubscription = createTransactionReemitter({
       genesisParameters$: this.genesisParameters$,
       logger: contextLogger(this.#logger, 'transactionsReemitter'),
+      // REVIEW: should this be a prop to SingleAddressWallet?
+      // Should default be computed?
+      reemitUnconfirmedAfterSlots: 300,
```

Comment:
I think we should reemit until we reach invalidHereafter. I can't think of any reason why we should stop reemitting if it wasn't confirmed.

Reply:
This will not make it stop re-emitting, it is a duration to wait until we consider that it was likely rolled back and re-emit. Perhaps the name of this param could be improved :sweat:

Reply:
🤦 . The name is clear. I was biased by my own expectations :))

Reply:
I would be for _not_ exposing it as configurable. Can we use the chainHistory polling interval as reference? If it's not there within 2 polling intervals, reemit

Reply:
Great idea! I think 2 might be too low, maybe 3 (or more) polling intervals :thinking:

Reply:
5c3a4167e5e6e870f89bed0fda5cf59895f35678